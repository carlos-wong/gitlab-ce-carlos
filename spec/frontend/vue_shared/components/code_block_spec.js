import { shallowMount } from '@vue/test-utils';
import CodeBlock from '~/vue_shared/components/code_block.vue';

describe('Code Block', () => {
  let wrapper;

  const createComponent = () => {
    wrapper = shallowMount(CodeBlock, {
      propsData: {
        code: 'test-code',
      },
    });
  };

  afterEach(() => {
    wrapper.destroy();
    wrapper = null;
  });

  it('matches snapshot', () => {
    createComponent();

    expect(wrapper.element).toMatchSnapshot();
  });
});
