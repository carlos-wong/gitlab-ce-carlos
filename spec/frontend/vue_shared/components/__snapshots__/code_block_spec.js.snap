// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code Block matches snapshot 1`] = `
<pre
  class="code-block rounded"
>
    
  <code
    class="d-block"
  >
    test-code
  </code>
  

</pre>
`;
